src/app/producer-shelf/components/producer-shelf/producer-shelf.component.html

```diff
@@ -1,12 +1,6 @@
-<div
-    *ngFor="let keyBinding of keyBindings" (click)="keyBinding.onMatched()"
-    class="keybinding"
-    [class.keybinding--matched]="isKeyBindingMatched(keyBinding)"
->
-    <span class="keybinding-label">{{ keyBinding.label }}</span>
-    <span class="keybinding-keys">{{ displayKeyBinding(keyBinding) }}</span>
+<div class="action-panels">
+    <sofie-tv2-action-panel *ngIf="cameraActions.length > 0" title="ADLIB PANEL: Camera (Scrollable)" [actions]="cameraActions" (executeAction)="executeAction($event)"></sofie-tv2-action-panel>
```

Comment:
I believe we still want to show the empty panels?
But it seems these are currently more for demo purporses untill we get the actual actions

Reply:
This is for demo purposes.